jmclient/jmclient/maker.py

```diff
@@ -262,7 +261,7 @@ def create_my_orders(self):
         """
 
     @abc.abstractmethod
-    def oid_to_order(self, cjorder, oid, amount):
+    def oid_to_order(self, cjorder, amount):
```

Comment:
Thanks for that catch. I recall seeing this unused field a year or two back in the *implementation* of this method and it got fixed in some commit or other ... I didn't realize that an abstract method like this could go unimplemented, or rather, that Python would allow an implementation (same name) with a different function signature.

Reply:
Really, almost everything in python that is not actual implementation is merely a suggestion :) Same goes for type hints and probably some more things. A IDE will, however, usually highlight such issues (as was the case for me).